ci/script.sh

```diff
@@ -36,7 +36,7 @@ case $TARGET in
     ;;
 
   integration)
-    RUST_MIN_STACK=8388608 cargo test --release --package runtime-integration-tests
+    RUST_MIN_STACK=8388608 cargo test --release --package runtime-integration-tests --features fast-runtime
```

Comment:
Required for newly added `pallet-block-rewards` integration tests which need to iterate at least four sessions (20 blocks/~2 min with `fast-runtime`)